### DISCUSSION TITLE
Sending notifications to database

### DISCUSSION THREAD

**Discussion by njugunamwangi** [filamentphp/filament#12055] [Help] [answered]
### Package

Notifications

### Package Version

v3

### How can we help you?

I recently started using Laravel 11 and I've had an issue with sending notifications to the database. It worked perfectly on Laravel 10 but on the new laravel version it doesnt seem to work. I used the correct approach: php artisan make:notifications-table for Laravel11, chained databaseNotifications() on the panel provider

**Comment by leandrocfe** [comment]
What is the issue?

**Reply by njugunamwangi** [reply]
->after(function(User $record) {
$recipients = User::role(Role::ADMIN)->get();

                        foreach($recipients as $recipient) {
                            Notification::make()
                                ->title($record->name . ' moved to ' . $record->stage->stage)
                                ->body(auth()->user()->name . ' moved ' . $record->name . ' to another stage')
                                ->icon('heroicon-o-puzzle-piece')
                                ->success()
                                ->sendToDatabase($recipient);
                        }
                    })
This here does not send the notification to the database

**Reply by micraux** [reply]
Same for me: worked perfectly with L10 but not with L11.

By the way, I noticed that with the imports and exports: not working anymore (written in DB but no file created).

I double checked everything but can't find any clue: don't know if it is related...

**Reply by njugunamwangi** [reply]
I kinda found the solution

run 'php artisan queue:work'

The default queue driver changed in Laravel 11

**Reply by micraux** [reply]
YES!!!
Thanks @njugunamwangi!

**Reply by njugunamwangi** [reply]
Keep learning!

**Reply by yuvrajtimalsina** [reply, marked as answer]
This seems to work as well :

```
QUEUE_CONNECTION=sync
```

**Reply by njugunamwangi** [reply]
Thay works too

**Reply by zineddine30** [reply]
this work perfectly so thank you

**Reply by mousa4007** [reply]
I wasted all day searching for a solution!! Why was this not mentioned in the documentation?

**Reply by yuvrajtimalsina** [reply]
> I wasted all day searching for a solution!! Why was this not mentioned in the documentation?

Maybe because the change was related to Laravel. 🫡

**Reply by daddybuggerr** [reply]
Thank you guys
